src/sdk/emitter/webAudioSoundEmitter.ts

```diff
@@ -8,34 +8,33 @@ import { SoundEmitter } from './soundEmitter';
  * @implements {SoundEmitter}
  */
 export class WebAudioSoundEmitter implements SoundEmitter {
```

Comment:
Please add another implementation called e.g. PromiseWebAudioSoundEmitter and that would be the previous version.